setup.py

```diff
@@ -18,8 +18,8 @@
     python_requires=">=3.7.0",
     install_requires=[
         "numpy>=1.18.0",
-        "exr-info @ git+https://github.com/Synthesis-AI-Dev/exr-info.git@v1.0.0#egg=exr-info-1.0.1",
-        "openexr @ git+https://github.com/jamesbowman/openexrpython.git#egg=openexr-1.3.2",
+        "exr-info @ git+https://github.com/Synthesis-AI-Dev/exr-info.git@v1.0.1#egg=exr-info",
+        "openexr @ git+https://github.com/jamesbowman/openexrpython.git#egg=openexr",
```

Comment:
be best to pin openexrpython to a specific revision since they don't use versions, i.e., @hash.. but can probably just ignore that if they're not changing it often

Reply:
Yeah, openexrpython has very few updates. I'd prefer to not pin it's version.